frontend/src/store/modules/configForm.js

```diff
@@ -245,6 +250,22 @@ export default {
       } catch (error) {
         context.commit('setExistingWebAdeConfig', `An error occurred getting the existing WebADE configuration for ${payload.acronym} in ${payload.webAdeEnv}`);
       }
+    },
+    async getAcronymDetails(context, acronym) {
+      try {
+        if (acronym) {
+          const response = await ApiService.getAcronymDetails(acronym);
+          if (!response) {
+            context.commit('setConfigSubmissionError', `An error occurred fetching application details for ${acronym}`);
+          }
+          context.commit('setSelectedAcronymDetails', response);
+        } else {
+          context.commit('setSelectedAcronymDetails', null);
+        }
+      } catch (error) {
+        context.commit('setConfigSubmissionError', `An error occurred fetching application details for ${acronym}`);
+      }
```

Comment:
Since this is already wrapped in a try catch, you can rely on the catch to do the right thing upon response failure.
```
      try {
        let response = null;
        if (acronym) {
          response = await ApiService.getAcronymDetails(acronym);
        }
        context.commit('setSelectedAcronymDetails', response);
      } catch (error) {
        context.commit('setConfigSubmissionError', `An error occurred fetching application details for ${acronym}`);
      }
```

Reply:
Fixed